test/client/rs.test.js

```diff
@@ -1,9 +1,9 @@
 describe('The RS implementation', function() {
   it('encodes HELLO as URYY\\. (ROT13 example)', function() {
-    assert.equal(RS.encode('HELLO'), 'URYY\\');
```

Comment:
might be nice to write a couple decode(encode(something)) == something test cases to last until we put something more thorough in place